letsencrypt-auto-source/pieces/bootstrappers/mac.sh

```diff
@@ -39,6 +39,6 @@ BootstrapMac() {
   if ! hash virtualenv 2>/dev/null; then
     say "virtualenv not installed."
     say "Installing with pip..."
-    pip install virtualenv
+    pip install --disable-pip-version-check virtualenv
```

Comment:
This change worries me a bit. It might be fine, however, we're assuming that the version of `pip` on the user's system is new enough to support `--disable-pip-version-check`. I don't think we can safely make this assumption and since we're deprecating our experimental `certbot-auto` support for macOS in #4943, I think we can just leave things as they've always been here.